x-pack/qa/saml-idp-tests/build.gradle

```diff
@@ -14,17 +12,17 @@ dependencies {
 testFixtures.useFixture ":x-pack:test:idp-fixture"
 
 String outputDir = "${project.buildDir}/generated-resources/${project.name}"
-def copyIdpFiles = tasks.register("copyIdpFiles", Copy) {
+def copyIdpFiles = tasks.register("copyIdpFiles", Sync) {
   from idpFixtureProject.files('idp/shibboleth-idp/credentials/idp-browser.pem', 'idp/shibboleth-idp/metadata/idp-metadata.xml',
     'idp/shibboleth-idp/credentials/sp-signing.key', 'idp/shibboleth-idp/credentials/sp-signing.crt');
   into outputDir
+  outputs.upToDateWhen { false } //ensure this copy is always done to prevent stale configuration
```

Comment:
This is kinda hacky ... I think I can fix this, but would prefer to do so on a different PR.

Reply:
I saw the comments in slack. I dont' think this is necessary. A sync task will also do a delete, followed by a copy so in what scenario would there be stale content here?

Reply:
I think what is happening is

First run: this task not upto date, copies the files, next task creates new file (with the same name) in the output directory 

Second run: this task reports up to date , even with sync because I ~think~ guess that the Sync task is only tracking the files it touches and did not touch the file was replaced by the other task (it has same name but it is a different file/inode), since it is up to date it does nothing and the elder file is still there and is used. 

I didn't spent too much time with this, so I could be wrong. It is pretty to reproduce by echoing out the contents of `$outputDir/idp-metadata.xml` in a doFirst of the setupPorts task.

Reply:
Yeah, that's quite odd. If one of the output files is modified it should trigger the task to be out-of-date, in which case, it would be the same behavior as just being never up to date as we have here. Kinda gross but yeah, I have plans to ditch all this build logic around test fixtures. It's positively awful.